harmonizer/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "harmonizer"
-version = "0.30.0"
+version = "0.31.0"
```

Comment:
This requires a run of at least `cargo test` or `build` to update the lock file.

Reply:
Good catch!